mainnet-1/specs/cosmoshub.json

```diff
@@ -0,0 +1,159 @@
+{
+  "proposal": {
+    "title": "Add Specs: Cosmos hub",
+    "description": "Adding new specification support for relaying cosmoshub data on Lava",
+    "specs": [
+      {
+        "index": "COSMOSHUB",
+        "name": "cosmos hub mainnet",
+        "enabled": true,
+        "imports": [
```

Comment:
when we import on mainnet we need to make sure all dependencies are also imported on mainnet and we don't have missing ones